ui/scripts/renderMockData.js

```diff
@@ -0,0 +1,76 @@
+const mapObject = require('./render-utils').mapObject
+
+function renderMockData ({ types }) {
+  return `// data is a tree organized by instanceId > zome > function
+// leaves can either be an object, or a function which is called with the zome call args.
+// See mockCallZome.js
+
+${mapObject(types, renderEntriesObject).join('\n')}
+
+const data = {
+  dna: {
+    zome: {
+${mapObject(types, renderZomeFunctionsForType).join('\n')}
+    }
+  }
+}
+
+export default data
+`
+}
+
+function renderEntriesObject (typeName) {
+  return `const ${typeName.toLowerCase()}Entries = {}`
+}
+
+function renderZomeFunctionsForType (typeName, fields) {
+  const name = typeName
+  const lowerName = name.toLowerCase()
+  const lowerNamePlural = name.toLowerCase() + 's'
+
+  return `      create_${lowerName}: ({ ${lowerName}_input: ${lowerName}Input }) => {
+        const ${lowerName}Index = Object.keys(${lowerName}Entries).length + 1
+        const id = 'Id' + ${lowerName}Index
+        const createdAt = String(Date.now())
+        ${lowerName}Entries[id] = { id, ...${lowerName}Input, created_at: createdAt }
+        return {
+          id,
+          created_at: createdAt,
```

Comment:
Not all entries will need a created_at.  Let's remove this as an assumed field in the entry, and just add it to the type-spec if we'd like it to be included.